addons/udes_priorities/models/priority.py

```diff
@@ -4,12 +4,14 @@
 from odoo.exceptions import UserError
 from uuid import uuid4
 
+
+MAIN_PROTECTED_FIELDS = {"reference"}
```

Comment:
`MAIN_PROTECTED_FIELDS` is a bit confusing. I can't think of a nicer way to link it with `PROTECTED_FIELDS` though...

Reply:
Agreed, maybe we could call this one BASIC_PROTECTED_FIELDS and the other one EXTENDED_PROTECTED_FIELDS?

Reply:
After much debate...

`MAIN_PROTECTED_FIELDS` -> `FORBIDDEN_FIELDS` and leave `PROTECTED_FIELDS` as they are.

I'd suggest adding a comment above each one to be clear that `FORBIDDEN_FIELDS` cannot be edited by anyone (not even admin), and `PROTECTED_FIELDS` can only be edited by trusted users.